src/team-extension.ts

```diff
@@ -317,11 +330,10 @@ export class TeamExtension  {
     }
 
     private ensureInitialized(): boolean {
-        if (!this._gitContext
-                || !this._gitContext.RemoteUrl
+        if (!this._repoContext
                 || !this._serverContext
                 || !this._serverContext.RepoInfo.IsTeamFoundation) {
-            this.setErrorStatus(Strings.NoGitRepoInformation);
+            this.setErrorStatus(Strings.NoRepoInformation);
             return false;
```

Comment:
Maybe check for the "expected repo type" here and have it passed in from the command. Just a thought.